src/game.c

```diff
@@ -322,6 +322,16 @@ static char *game_version(const char *path, const char *d_name)
 	trunc_lines(p, 0);
 	return p;
 }
+
+static char *game_lang(const char *path, const char *d_name)
+{
+	char *p = game_tag(path, d_name, "Language");
+	if (!p)
+		return strdup(d_name);
```

Comment:
Here is, if there is no tag in main* file, it returns directory name... It's strange... May be better is to return NULL?

Reply:
Yes, that's better, changed it.